test/linter/test_gitleaks.vader

```diff
@@ -0,0 +1,37 @@
+Before:
+  call ale#assert#SetUpLinterTest('tex', 'gitleaks')
```

Comment:
Let's add in a test case with the supported filetypes in a List where we check if the linter is set up for all of the filetypes we need in a loop. We can write one filetype per line. Calling the tear down and set up functions in a loop will probably work.